README.md

```diff
@@ -14,6 +14,9 @@ UCSD DAMS Vagrant Virtual Machine
 
 You can shell into the machine with `vagrant ssh` or `ssh -p 2222 vagrant@localhost`
 
+You can load test records into DAMS Repository, installed in the Vagrant VM, using cURL::
+`curl -u dams:dams -d @your_damspas_path/damspas/spec/fixtures/damsObject.xml http://localhost:8080/dams/api/objects/bd22194583`
```

Comment:
I really like how you merged the example fixtures with the curl command, nice!

Sorry for more formatting requests, but can you please:

1. Remove the extra colon (:)
2. Force the curl code snippet onto its own line? It's wrapping as an inline block and might be hard for people to easily process.